cli/chaosli/README.md

```diff
@@ -0,0 +1,59 @@
+# Chaosli
+
+Chaosli (Chaos Line Interface) is motivated by creating a more user friendly and digestible chaos-controller experience. The CLI pairs with your Disruption configuration, giving you information to give you a better sense of what is going on or what will happen when you apply new disruptions. With features like explain and validation, it gives users, not only a better understanding, but a better experience with the controller.  
+
+
+#### Table of Contents
+---
+- Validate
+- Explain
+- Create
+
+#### Validate
+---
+Usage: `go run chaosli/main.go validate --path <path to disruption file>`
+
+Description: Validates your disruption files to make sure everything is correctly formatted.
+
+Example:
+
+```
+# Running Validate on a Configuration that has no Disruptions specified
+# Note; We removed the network portion from the actual example for this example, normally
+# network_delay.yaml in examples has a network disruption specified
+
+$ go run chaosli/main.go validate --path=../examples/network_delay.yaml
+Error: cannot apply an empty disruption - at least one of Network, DNS, DiskPressure, NodeFailure, CPUPressure fields is needed
+```
+
+#### Explain
+---
+Usage: `go run chaosli/main.go explain --path <path to disruption file>`
+
+Description: Prints out a summary of the disruption defined by `path`.
```

Comment:
I think we should try to have consistency across `validate` and `explain` with how we refer to `path` in the description

Reply:
Pushing a new commit, let me know if my change isnt what you meant.